### PR TITLE
tweak the org plugin to better use builtin facilities

### PR DESCRIPTION
**Pull request by freben**
Just some freshin' up

### CODE REVIEW

File: plugins/api-docs/src/components/ApiDefinitionCard/ApiDefinitionCard.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ type Props = {
 };
 
 export const ApiDefinitionCard = (_: Props) => {
-  const entity = useEntity().entity as ApiEntity;
+  const entity = useEntity<ApiEntity>().entity;
```
Comment (Fox32) [inline plugins/api-docs/src/components/ApiDefinitionCard/ApiDefinitionCard.tsx:31]: This is something I didn't know till now! How about
```suggestion
  const {entity} = useEntity<ApiEntity>();
```

Comment (Fox32) [reply plugins/api-docs/src/components/ApiDefinitionCard/ApiDefinitionCard.tsx:31]: How can I know about it 🤦 You just implemented it in this PR.... 😆

File: plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx [hunk 2]
```diff
@@ -83,23 +55,25 @@ export const GroupProfileCard = ({
   entity?: GroupEntity;
   variant?: InfoCardVariants;
 }) => {
-  const group = useEntity().entity as GroupEntity;
+  const group = useEntity<GroupEntity>().entity;
+  if (!group) {
+    return <Alert severity="error">User not found</Alert>;
```
Comment (dhenneke) [inline plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx:60]:
```suggestion
    return <Alert severity="error">Group not found</Alert>;
```

Comment (freben) [reply plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx]: fix'd!

File: plugins/api-docs/src/components/ApiDefinitionCard/ApiDefinitionCard.tsx (hunk 1 above)
Comment (freben) [reply plugins/api-docs/src/components/ApiDefinitionCard/ApiDefinitionCard.tsx]: Applied that pattern all over now though (destructuring)

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**